src/CMakeLists.txt

```diff
@@ -170,6 +170,10 @@ macro (add_compiler_flag flag)
   set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${flag}")
 endmacro (add_compiler_flag)
 
+# Disallow undefined symbols in object files. Undefined symbols in shared
+# libraries are still allowed.
```

Comment:
Hmm, I am a bit confused by the comment, isn't `CMAKE_SHARED_LINKER_FLAGS` used for shared libraries?
And aren't object files just archives and hence have always undefined symbols?